packages/graphql-server/src/functions/graphql.ts

```diff
@@ -2,26 +2,20 @@
 import { useDepthLimit } from '@envelop/depth-limit'
 import { useDisableIntrospection } from '@envelop/disable-introspection'
 import { useFilterAllowedOperations } from '@envelop/filter-operation-type'
-import type { PluginOrDisabledPlugin } from '@graphql-yoga/common'
-import {
-  EnvelopError,
-  FormatErrorHandler,
-  GraphQLYogaError,
-} from '@graphql-yoga/common'
-import { createServer } from '@graphql-yoga/common'
 import type {
   APIGatewayProxyEvent,
   APIGatewayProxyResult,
   Context as LambdaContext,
 } from 'aws-lambda'
-import { Headers, Request } from 'cross-undici-fetch'
 import { GraphQLError, GraphQLSchema, OperationTypeNode } from 'graphql'
+import { Plugin, MaskError, useReadinessCheck } from 'graphql-yoga'
```

Comment:
@ardatan I upgraded graphql-yoga to v3.1 on my fork and it looks like it doesn't export `MaskError` anymore—we should upgrade the versions on this PR right? (I'm happy to do so if it's ok.) And do you know where that type should come from now?

Reply:
Maybe @envelope/core? https://github.com/dotansimha/graphql-yoga/blob/main/packages/graphql-yoga/src/types.ts#L1

Reply:
You're right! I updated the code!